operator/v1/0000_100_vsphere_problem_detector_01_config.crd.yaml

```diff
@@ -0,0 +1,175 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
```

Comment:
is this deployed on every platform? Having this on IBM Cloud or AWS will look strange.

Reply:
removed